presto-docs/src/main/sphinx/connector/phoenix.rst

```diff
@@ -0,0 +1,153 @@
+=================
+Phoenix Connector
+=================
+
+The Phoenix connector allows querying data stored in Phoenix.
```

Comment:
Do you refer https://phoenix.apache.org/? If so then I believe you should use Apache Phoenix.